modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js

```diff
@@ -22,40 +22,14 @@
 
 var operations = '.wr-operations',
     modalPopup = '.wr-modalpopup',
-    // modalPopupContainer = modalPopup + ' .modalpopup-container',
     modalPopupContent = modalPopup + ' .modalpopup-content',
-    // deviceCheckbox = '#ast-container .ctrl-wr-asset .itm-select input[type="checkbox"]',
-    // showOperationsBtn = '#showOperationsBtn',
     navHeight = $('#nav').height(),
     headerHeight = $('header').height(),
     offset = (headerHeight + navHeight),
-    // maxOperationsLimit = 15,
-    // hiddenOperation = '.wr-hidden-operations-content > div',
-    deviceSelection = '.device-select';
-    // dataTableSelection = '.DTTT_selected',
-    // currentOperationList = [];
-
-/*
-* On window resize functions.
-*/
-//$(window).resize(function(){
-//    toggleMoreOperationsHeight();
-//});
-
-/*
- * On main div.container resize functions.
- * @required  jquery.resize.js
- */
-//$('.container').resize(function(){
-//    toggleMoreOperationsHeight();
-//});
-
-/*
- * On Show Operations click operation show toggling function.
- */
-//function showOperations(){
-//    $(operations).toggle('slide');
-//}
+    deviceSelection = '.device-select',
+    IOS = 'ios',
+    WINDOWS = 'windows',
+    ANDROID = 'android';
```

Comment:
for defining constants, better using an approach like this
var platformTypeConstants = {
    "ANDROID": "android",
    "IOS": "ios",
    "WINDOWS": "windows"
};